### ISSUE TITLE
Using the "outDir" option on "assets" configuration does nothing

### ISSUE THREAD

**Issue by renanhangai** [nestjs/nest-cli#561]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

The `"assets"` compiler option does not respect the `"outDir"`. When using the "outDir" option, the compiler ignores it and copies the assets to the default outDir (`"dist"`) folder.

## Expected behavior

According to the docs, it should copy the assets to "outDir"
https://docs.nestjs.com/cli/monorepo#assets
![image](https://user-images.githubusercontent.com/1877289/73902293-18574480-4874-11ea-8711-84e4a4d21d4f.png)

## Minimal reproduction of the problem with instructions

Create a  `example.graphql` file inside the `src` dir

Use the following `nest-cli.json`
```json
{
	"collection": "@nestjs/schematics",
	"sourceRoot": "src",
	"compilerOptions": {
		"assets": [{ 
			"include": "**/*.{gql,graphql}", 
			"outDir": "dist/output/graphql" 
		}]
	}
}
```

When running any of the `nest` command, the file should be copied to `dist/output/graphql/example.graphql`, but currently it is being copied to `dist/example.graphql`

## Environment

```
[System Information]
OS Version     : Linux 5.3
NodeJS Version : v12.12.0
YARN Version    : 1.19.1 

[Nest CLI]
Nest CLI Version : 6.14.1 

[Nest Platform Information]
platform-express version : 6.10.14
graphql version          : 6.5.4
typeorm version          : 6.2.0
common version           : 6.10.14
core version             : 6.10.14
```

It looks like the PR #515 already solves the problem, but i think there was a misunderstanding of what the author was trying to say.

**Comment by kamilmysliwiec** [comment]
Fixed in 6.14.2. Thanks for reporting.

**Comment by j-ibarra** [comment]
hello

what do you think if in the outDir include a var for example <rootDir>

currently I have a mono repo whit config folder in each app, if I declare the config folder into src all work fine, but if i declare the config folder in same level that src, I need used include and outDir, but declaring all ourDir peer app, It would be great to be able to declare the variable globally in the outDir, for example


folder structure

```
/apps
    /app1
        /config
            default.yml
            local.yml
            production.yml
        /src
    /app2
        /config
            default.yml
            local.yml
            production.yml
        /src
```

```
"compilerOptions": {
        "webpack": true,
        "assets": [
            {
                "include": "../config/*.yml",
                "outDir": "dist/<rootDir>/config"
            }
        ]
    }
```
